lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp

```diff
@@ -6754,12 +6754,12 @@ size_t TypeSystemClang::GetIndexOfChildMemberWithName(
           llvm::StringRef field_name = field->getName();
           if (field_name.empty()) {
             CompilerType field_type = GetType(field->getType());
+            std::vector<uint32_t> save_indices = child_indexes;
             child_indexes.push_back(child_idx);
             if (field_type.GetIndexOfChildMemberWithName(
                     name, omit_empty_base_classes, child_indexes))
               return child_indexes.size();
-            child_indexes.pop_back();
-
+            child_indexes = save_indices;
```

Comment:
Whichever is the resolution for the discussion above, we should at the very least `child_index = std::move(save_indices)` here

Reply:
Done.